src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number | string) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastPage = Math.ceil(total / perPage);
+  const amountOfPage = getNumbers(1, lastPage);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames(
+            'page-item',
+            { disabled: currentPage === 1 },
+          )}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={() => {
+              if (currentPage !== 1) {
+                onPageChange('prev');
```

Comment:
Instead of calling `onPageChange('prev')`, consider calling `onPageChange(currentPage - 1)` for better clarity and consistency with the function's argument type.